### DISCUSSION TITLE
What's the branch with the latest version?

### DISCUSSION THREAD

**Discussion by zion830** [euphony-io/euphony#135] [Q&A] [answered]
Hi! I added euphony version `0.7.1.6` to my gradle project.
I read the guide in README and used ExTxManager. 
And I found out that the downloaded source code is different from the origin source code in this repository. 

For example, the function `euInitTransmit()` is shown as deprecated. But I can't find the new function in my local environment.

So my questions are...
* What is the branch or commit of the latest version?
* Can I still use this function?

**Comment by designe** [comment, marked as answer]
Hi! @zion830 
For your 1st & 2nd question, euphony's latest version is on the master branch.
Actually, `euInitTransmit()` is deprecated. But it works well for latest version.
https://github.com/euphony-io/euphony/blob/71825239ba14b70cc1029404b0b640adc2e0a0f9/euphony/src/main/java/co/euphony/tx/EuTxManager.java#L143
it is replaced by `setCode` function. so, you can use like below.
```kotlin
EuTxManager mTxManager = new EuTxManager();
mTxManager.setCode("Hello, Euphony"); // To generate acoustic data "Hello, Euphony"
```

**Reply by designe** [reply]
And.. it means that we need to update `README.md` using `setCode()` function.
Thanks for checking out the documentation that hasn't been updated.
Could you make a PR for this?

**Reply by zion830** [reply]
yes :D thank you for your answer!
